browser.js

```diff
@@ -19,16 +19,16 @@ exports.createImageData = function (array, width, height) {
 }
 
 exports.loadImage = function (src) {
-  return new Promise((resolve, reject) => {
+  return new Promise(function (resolve, reject) {
     const image = document.createElement('img')
 
     function cleanup () {
       image.onload = null
       image.onerror = null
     }
 
-    image.onload = () => { cleanup(); resolve(image) }
-    image.onerror = () => { cleanup(); reject(new Error(`Failed to load the image "${src}"`)) }
+    image.onload = function () { cleanup(); resolve(image) }
+    image.onerror = function () { cleanup(); reject(new Error('Failed to load the image ' + src + '"')) }
```

Comment:
missing double quote "
should be `image "' +`

Reply:
@thisconnect Fixed, thank you!